include/openssl/opensslconf.h

```diff
@@ -48,6 +48,7 @@
 #define OPENSSL_NO_MD2
 #define OPENSSL_NO_MDC2
 #define OPENSSL_NO_OCB
+#define OPENSSL_NO_PHA
```

Comment:
Should we name this to something clearer? Maybe something like 
```suggestion
#define OPENSSL_NO_TLS1_3_PHA
```

Reply:
this makes sense today, but PHA will likely be present in TLS versions >1.3 as well

Reply:
Sure that makes sense. How about something like `OPENSSL_NO_TLS_PHA`?
The list of `OPENSSL_NO_*` macros tend to be individual algorithms and protocols. Since `PHA`'s a concept defined within the TLS protocol, I feel like there's a need to be more specific.

Reply:
agreed. updated.